Dockerfile

```diff
@@ -1,49 +1,50 @@
-FROM	debian:wheezy
-
+FROM	   node:6.7.0-slim
```

Comment:
I did not find anything that required them. Did you have something in mind @gwwar?